packages/rolldown/src/utils/transform-side-effects.ts

```diff
@@ -0,0 +1,30 @@
+import { ModuleSideEffects } from '../plugin'
+
+// TODO The typing should import from binding, but const enum is disabled by `isolatedModules`.
```

Comment:
@Brooooooklyn  It there a way to generate non-const enum in NAPI-RS for enum?

Reply:
`napi build --no-const-enum`